### DISCUSSION TITLE
Vuejs whit Symfony

### DISCUSSION THREAD

**Discussion by Neyto-1** [symfony/symfony#48073] [Q&A]
Hi, do you know how to implement Vue JS in Symfony? Being on Ubuntu

**Comment by jakubtobiasz** [comment]
Hi @Neyto-1 🙋🏼‍♂️!
You can consider using Symfony UX 👉🏼 https://ux.symfony.com/vue.

**Comment by chyshyya** [comment]
https://symfony.com/doc/current/frontend/encore/vuejs.html
